mobx/lib/src/api/extensions/observable_map_extension.dart

```diff
@@ -0,0 +1,6 @@
+part of '../extensions.dart';
+
+/// Turn the Map into an ObservableMap.
+extension ObservableMapExtension<K, V> on Map<K, V> {
+  ObservableMap<K, V> asObservable() => ObservableMap<K, V>.of(this);
```

Comment:
Let's include named, optional context parameters for all `asObservable` methods.

Reply:
Like how you're doing it with `Future`s and `Stream`s.

Reply:
Added!